features/custom_matchers/define_diffable_matcher.feature

```diff
@@ -27,3 +27,49 @@ Feature: define diffable matcher
              +two
               lines
       """
+
+  Scenario: Redefine actual
+
+    Sometimes is neccessary to overwrite actual to make diffing work, e.g. if
+    `actual` is a name of a file you want to read from. For this to work you
+    need to overwrite `@actual` in your matcher.
```

Comment:
Does cucumber support prose between the scenario title and the first step like this?  Even if cucumber supports it, I'm not sure how it'll look on relish...